### ISSUE TITLE
make fails on Arch Linux

### ISSUE THREAD

**Issue by egore** [alexlarsson/spotify-app#4]
I followed the instructions but building spotify-app on Arch Linux fails:

`
/usr/bin/g-ir-scanner  --warn-all --add-include-path=. --namespace=GConf --nsversion=2.0 --libtool="/bin/sh ../libtool"  --include=GObject-2.0 --pkg-export=gconf-2.0  --library=libgconf-2.la --identifier-prefix=GConf --symbol-prefix=gconf --c-include "gconf/gconf.h" -I.. -I.. -pthread -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -I/usr/include/libxml2 -I/app/include/dbus-1.0 -I/usr/include/dbus-1.0 -I/usr/lib/dbus-1.0/include -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -DG_LOG_DOMAIN=\"GConf\" -DPREFIX=\""/app"\" -DGCONF_LOCALE_DIR=\""/app/share/locale"\" -DGCONF_SRCDIR=\""/run/build/gconf"\" -DGCONF_CONFDIR=\""/app/etc/gconf/2"\" -DGCONF_ETCDIR=\""/app/etc/gconf"\" -DGCONF_BINDIR=\""/app/bin"\" -DGCONF_SERVERDIR=\""/app/libexec"\" -DGCONF_BUILDDIR=\"".."\" -DGCONF_BACKEND_DIR=\""/app/lib/GConf/2"\" -DVERSION=\""3.2.6"\" -DGCONF_ENABLE_INTERNALS=1 -DGCONFD=\""gconfd-2"\"   gconf.h gconf-changeset.h gconf-listeners.h gconf-schema.h gconf-value.h gconf-error.h gconf-engine.h gconf-client.h gconf-enum-types.h gconf-internals.c gconf-backend.c gconf-changeset.c gconf-error.c gconf-listeners.c gconf-locale.c gconf-schema.c gconf-sources.c gconf-value.c gconf.c gconf-client.c gconf-enum-types.c gconf-dbus.c gconf-dbus-utils.c libgconf-2.la Makefile --output GConf-2.0.gir
/usr/share/gobject-introspection-1.0/Makefile.introspection:150: recipe for target 'GConf-2.0.gir' failed
make[3]: *** [GConf-2.0.gir] Illegal instruction (core dumped)
make[3]: *** Waiting for unfinished jobs....
`

gobject-introspection: 1.48.0-1
flatpak: 0.6.5-1

**Comment by alexlarsson** [comment]
Wot? illegal instruction? Is this repeatable?

**Comment by egore** [comment]
Sadly it's reproducible. Building the sources of gconf manually works fine. Find attached the build logs. I'm out of ideas why it would fail.

[configure && make](https://github.com/alexlarsson/spotify-app/files/328548/20160622_configure_make.txt)
[flatpak-builder](https://github.com/alexlarsson/spotify-app/files/328549/20160622_flatpak_make.txt)

**Comment by alexlarsson** [comment]
This seems related to https://github.com/flatpak/flatpak/issues/143
Both are getting weird illegal instructions.

**Comment by alexlarsson** [comment]
Yes, this seems to be the same thing. They are both running python2 apps (waf, and g-ir-scanner) and dying from illegal instruction.
